docs/guides/custom-redirects.mdx

```diff
@@ -148,7 +156,11 @@ function App() {
 
 ### Button components
 
-In the case that you are using a [`<SignInButton>`](/docs/components/unstyled/sign-in-button) or [`<SignUpButton>`](/docs/components/unstyled/sign-up-button), you can also pass the properties into those components. We recommend defining both `signInFallbackRedirectUrl` and `signUpFallbackRedirectUrl` redirects in each button as some users may choose to sign up instead after attempting to sign in.
+[`<SignInButton>`](/docs/components/unstyled/sign-in-button) and [`<SignUpButton>`](/docs/components/unstyled/sign-up-button) also take these properties.
```

Comment:
Can we also add this copy to the "Control components" section as well for consistency?

```suggestion
[`<SignInButton>`](/docs/components/unstyled/sign-in-button) and [`<SignUpButton>`](/docs/components/unstyled/sign-up-button) also accepts these properties.
```